libs/composer/composer.json

```diff
@@ -29,7 +32,9 @@
 		"mockery/mockery": "^0.9.9"
 	},
 	"autoload": {
-		"psr-4" : { "ILIAS\\" : "../../src" }
+		"psr-4" : { "ILIAS\\" : "../../src" },
+		"classmap": ["../../Services/", "../../Modules"],
+		"exclude-from-classmap": ["../../Services/Migration", "../../*/*/lib", "../../*/*/test", "../../Services/CAS/phpcas"]
```

Comment:
@smeyer-ilias Something a bit of-topic: I found an external library in Services/CAS/phpcas which is not in the subdir Services/CAS/libs/... . The phpcas also could be added using composer. If you have the time, would be cool to move it